### PR TITLE
Week4

### PR DESCRIPTION
**Pull request by Maarimar**

### CODE REVIEW

File: 02-express-tutorial/controllers/people.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+const createPersonPostman = (req, res) => {
+  const { name } = req.body
+  if (!name) {
+    return res
+      .status(400)
+      .json({ success: false, msg: 'please provide name value' })
+  }
+  res.status(201).json({ success: true, data: [...people, name] })
```
Comment (jdevries3133) [inline 02-express-tutorial/controllers/people.js:24]: What's the difference between status 201 and status 200?

File: 02-express-tutorial/routes/auth.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express.Router();
+
+
+app.post('/',(req,res)=>{
```
Comment (jdevries3133) [inline 02-express-tutorial/routes/auth.js:5]:
```suggestion
app.post('/', (req,res) => {
```

Some more formatting items:

- Always put a space after commas in function calls & arrays
- Put spaces around the arrow (it's considered an operator just like `+`, `-`, `=`, etc.

**Review by jdevries3133 — APPROVED** [review]
Looks good, nice work!

**Comment by jdevries3133** [comment]
Actually, I think you are missing the additional "practice middleware" assignment. Take a look at the assignment page: https://learn.codethedream.org/ctd-node-express-class-lesson-4-middleware-rest-methods-and-postman/ at the section titled, "Additional Assignment: Practice Middleware"

File: 02-express-tutorial/09-middleware-use.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+const express = require("express");
+const app = express();
+const authorized = (req, res, next) => {
+    const {user}=req.query;
```
[the remark this answers is not in the document]
Comment (Maarimar) [reply 02-express-tutorial/09-middleware-use.js:4]: Thank you so much for the advice! I am going to start using prettier :)

File: 02-express-tutorial/11-methods.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+
+// statis asset
+
+app.use(express.static('./methods-public'))
+
+//parse from data
+app.use(express.urlencoded({extended:false}))
+
+// parse jason
```
[the remark this answers is not in the document]
Comment (Maarimar) [reply 02-express-tutorial/11-methods.js:15]: haha geez that was bad

File: 02-express-tutorial/controllers/people.js (hunk 1 above)
Comment (Maarimar) [reply 02-express-tutorial/controllers/people.js:24]: 200 means we got the information and there is no errors and 201 is for PUT requests meaning something was created.

**Pull request merged**